rideshare.rb

```diff
@@ -0,0 +1,143 @@
+drivers = {
+  DR0001: [
+    {
+      date: '02/03/2016',
+      cost: 10,
+      rider_ID: 'RD0003',
+      rating: 3
+    },
+    {
+      date: '02/03/2016',
+      cost: 30,
+      rider_ID: 'RD0015',
+      rating: 4
+    },
+    {
+      date: '02/05/2016',
+      cost: 45,
+      rider_ID: 'RD0003',
+      rating: 2
+    }
+  ],
+  
+  DR0002: [
+    {
+      date: '02/03/2016',
+      cost: 25,
+      rider_ID: 'RD0073',
+      rating: 5
+    },
+    {
+      date: '02/04/2016',
+      cost: 15,
+      rider_ID: 'RD0013',
+      rating: 1
+    },
+    {
+      date: '02/05/2016',
+      cost: 35,
+      rider_ID: 'RD0066',
+      rating: 3
+    }
+  ],
+  
+  DR0003: [
+    {
+      date: '02/04/2016',
+      cost: 5,
+      rider_ID: 'RD0066',
+      rating: 5
+    },
+    {
+      date: '02/05/2016',
+      cost: 50,
+      rider_ID: 'RD0003',
+      rating: 2
+    }
+  ],
+  
+  DR0004: [
+    {
+      date: '02/03/2016',
+      cost: 5,
+      rider_ID: 'RD0022',
+      rating: 5
+    },
+    {
+      date: '02/04/2016',
+      cost: 10,
+      rider_ID: 'RD0022',
+      rating: 4
+    },
+    {
+      date: '02/05/2016',
+      cost: 20,
+      rider_ID: 'RD0073',
+      rating: 5
+    }
+  ]
+}
+
+puts "TOTAL RIDES PER DRIVER:"
+drivers.each_key do |driver|
+  puts "Driver #{driver} has given #{drivers[driver].count} rides."
+end
+
+puts "\nTOTAL AMOUNT MADE PER DRIVER:"
+mostMoney = 0
```

Comment:
Remember, in Ruby we use snake_case for variables, so this should read `most_money = 0`.